crates/driver/src/tests/cases/settle.rs

```diff
@@ -73,33 +76,24 @@ async fn private_rpc_with_high_risk_solution() {
     err.kind("FailedToSubmit");
 }
 
-// /// Checks that we can settle transactions that have a gas limit higher than
-// /// half the block size
-// #[tokio::test]
-// #[ignore]
-// async fn high_gas_limit() {
-//     let test = tests::setup()
-//         .name("high gas limit")
-//         .pool(ab_pool())
-//         .order(ab_order())
-//         .solution(ab_solution().increase_gas(15_000_000))
-//         .done()
-//         .await;
-
-//     // Set to 30M for solving purposes
-//     test.web3()
-//         .transport()
-//         .execute("evm_setBlockGasLimit", vec![serde_json::json!(30_000_000)])
-//         .await
-//         .unwrap();
+/// Checks that very large settlements are rejected by the driver.
```

Comment:
Should we also test that submitting very big solution that are still under the limit still works?
Otherwise we could theoretically not get a solution here for any number of reasons.